tests/subsys/dfu/dfu_target_stream/pm_config.h

```diff
@@ -0,0 +1,7 @@
+/* generated file copied to simplify building the test */
```

Comment:
Would it be more "correct" to add a pm_static.yml instead?

Reply:
Its not needed any more actually. This was left over from a previous version of dfu_target_stream.

Reply:
Deleted the file, its not needed any more.